# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tibit/ggrab/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./pesstream.h.orig	2004-09-12 05:05:58.767961504 +0200
+++ ./pesstream.h	2004-09-12 05:06:13.109781216 +0200
@@ -25,7 +25,7 @@
 	private:
 		static int		m_st_nr;
 		int			m_nr;
-		enum S_TYPE		m_stype;
+		S_TYPE		m_stype;
 		class CBuffer *		mp_cbuf;
 		class xlist *		mp_list;
 
